src/components/common/program-section-templates/faq-program-section/FaqProgramSection.module.scss

```
@@ -0,0 +1,136 @@
+    }
+
+    .editable & {
+        border-top: none;
+        gap: 12%;
+    }
+}
+
+.title {
+    flex: 0 0 41%;
+    @include type.h3-bold;
+    text-transform: uppercase;
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)